### PR TITLE
Simplify a bit documentation and use correct names in example

### PR DESCRIPTION
**Pull request by antban**
Simplify a bit documentation and use correct names in example

### CODE REVIEW

**Comment by antban** [comment]
:+

**Comment by antban** [comment]
:+1:

File: docs/_documentation/using_per_event_authorization.md [hunk 1]
```diff
@@ -25,30 +25,18 @@ Nakadi provides per-event filtering, allowing event type publishers to specify w
...
-      "exclusive_readers": "Team-A"
-    }
-    "metadata": {
-      "eid": "d765de34-09c0-4bbb-8b1e-7160a33a0791",
-      "occurred_at": "2016-03-15T23:47:15+01:00"
-    }
-  }
-```
+ The events that were published to the event type above could be read only by a set of readers, that has matching 
+ `retailer_id` provided by authorization plugin. In case if consumer do not have this value - the events are silently
```
Comment (adyach) [inline docs/_documentation/using_per_event_authorization.md:38]: consumer does

**Comment by adyach** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**